### ISSUE TITLE
Create Infra Mapping Wizard does not show all Clusters/Datastores/Networks

### ISSUE THREAD

**Issue by kedark3** [ManageIQ/miq_v2v_ui_plugin#244]
Create Infra Mapping Wizard does not show all Clusters/Datastores/Networks when we add more than one VMware/RHV providers to ManageIQ Appliance. 
Steps:
1. Create a new Appliance
2. Add 2 VMware providers
3. Add 2 RHV providers
4. Check Migration Infra Mapping wizard

Result: Cannot see all Clusters/Networks/Datastores in the wizard.
Expected: Should show all the items correctly. 

https://bugzilla.redhat.com/show_bug.cgi?id=1571402

**Comment by bzwei** [comment]
The description is not sufficient for debugging. We should get data from all providers. Need to access the actual environment with more details - what resources are expected but not showing.

**Comment by kedark3** [comment]
@bzwei @vconzola I did validate this again with the latest appliance. It seems to be working fine. Only issue I found is with the "Clusters" section in "Infrastructure Mapping Wizard". What happens is, for my VMware and RHV providers(I added 2 of each) I can see exact same name in the Clusters because our our clusters on All RHV providers and All VMware providers are named exactly. So I wish we have a way to add Provider name next to cluster names or some other way to differentiate. 

Here's a screenshot:
![image](https://user-images.githubusercontent.com/8180469/40191941-dd800790-59d0-11e8-8866-2c56067f699b.png)
